swagger_server/controllers/samples_get_ids_controller.py

```diff
@@ -0,0 +1,32 @@
+from swagger_server.db import get_db
+from swagger_server.exceptions import NotFound
+from swagger_server.models.error import Error  # noqa: E501
+from swagger_server.ogm.mappers import SampleNode
+
+
+def samples_get(ids=None):  # noqa: E501
+    """samples_get
+
+    Return one or more samples based on IDs # noqa: E501
+
+    :param ids: A comma-separated list of sample IDs
+    :type ids: List[str]
+
+    :rtype: List[Sample]
+    """
+
+    sample_graph = get_db()
+
+    samples = []
+    for sample_id in ids:
+        try:
+            node = SampleNode.get(sample_id, sample_graph)
```

Comment:
I think this makes multiple queries? How about https://py2neo.org/v4/ogm.html#object-matching?

Reply:
Yes, that will make multiple queries, which is not good for performance. The object matching looks good and I have used that instead in 08fb750. The tests did speed up quite a lot.

Thanks for pointing this out.